src/auth0-sso-login.js

```diff
@@ -179,6 +180,10 @@ export default class auth {
                     });
                   }
                 });
+              })
+              .catch((error) => {
+                this.log('Error while calling renewAuth after successful Auth0Lock authentication', error);
```

Comment:
Same with this one.

Reply:
Hmm, it turns out we are totally inconsistent, sometimes logging a `String` others an `Object` (i.e. Error), and this others `(String, Object)`.

Maybe I'll clean that in the auth0 library upgrade.